api/oc_main.c

```diff
@@ -225,6 +232,12 @@ oc_main_init(const oc_handler_t *handler)
     oc_shutdown_all_devices();
     goto err;
   }
+#ifdef OC_DYNAMIC_ALLOCATION
+  drop_commands = (bool *)calloc(1, sizeof(bool));
```

Comment:
Is allocating memory just for one bool sufficient, shouldn't it be

```
calloc(OC_MAX_NUM_DEVICES, sizeof(bool));
```

?

Reply:
Fixed